Documentation Updates related to Range.values (#5821)
OfficeDev/office-js-docs-pr: docs/concepts/resource-limits-and-performance-optimization.md

Excel add-ins have important data transfer limits when interacting with the workbook.

- Excel on the web has a payload size limit for requests and responses of **5MB**. `RichAPI.Error` will be thrown if that limit is exceeded.

> - A range is limited to **5,000,000** cells for read operations. When a read exceeds this limit, the API might return `null` instead of throwing an error. For more information, see [Payload size limit best practices](../excel/performance.md#payload-size-limit-best-practices).

Reviewer: ```suggestion
- A range is limited to **5,000,000** cells for read operations. When a read exceeds this limit, the API will either return `null` or throw an error. For more information, see [Payload size limit best practices](../excel/performance.md#payload-size-limit-best-practices).
```
